pkgs/applications/networking/protonvpn-cli-ng/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub, python, python3Packages, openvpn, dialog }:
+
+with stdenv.lib;
```

Comment:
I think this is superfluous, because you only need it where you do it also at `meta`
```suggestion
```